new_dawn_server/users/api/tests/test_users_api.py

```diff
@@ -45,28 +45,25 @@ def test_register_success(self):
         }
         res = self.api_client.post(
             "/api/v1/register/", format="json", data=all_arguments)
+
+        user = User.objects.get(username="test-user")
         res_data = json.loads(res.content)
-        for k, v in all_arguments.items():
-            self.assertTrue(k in res_data)
-            if k == "password":
-                self.assertTrue(check_password(v, res_data[k]))
-            else:
-                self.assertEqual(v, res_data[k])
+        res_data["username"] = self.register_arguments["username"]
```

Comment:
@tangziyi001 Why do you use two lines to overwrite since "username" and "token" are already in response?